website/templates/log_list.mako

```diff
@@ -54,8 +54,19 @@
                                         <span class="overflow" data-bind="text: log.userFullName"></span>
                                     </span>
                                 </span>
-                                <!-- Log actions are the same as their template name -->
-                                    <span data-bind="template: {name: log.action, data: log}"></span>
+                                <!-- Handle specific language for contributor bibliographic/non-bibliographic on anonymous VOLs -->
+                                    <!-- ko if: log.action == 'made_contributor_visible' && log.anonymous -->
```

Comment:
This doesn't look like the right place for this logic. Can you put this logic into `log_templates.mako` instead?